src/Microsoft.VisualStudio.LanguageServices.Razor/Editor/DefaultVisualStudioDocumentTracker.cs

```diff
@@ -79,6 +78,10 @@ internal class DefaultVisualStudioDocumentTracker : VisualStudioDocumentTracker
 
         public override Workspace Workspace => _workspace;
 
+        public override string FilePath => _filePath;
```

Comment:
Should we just pass in the filename on construction?

Reply:
If we eventually want to control what happens when a file moves we'll need to have direct control of the file path.